tests/Nox.Types.Tests/Types/DateTimeRange/DateTimeRangeTests.cs

```diff
@@ -0,0 +1,322 @@
+﻿using System.Globalization;
+
+namespace Nox.Types.Tests.Types;
+
+public class DateTimeRangeTests
+{
+    [Fact]
+    public void Nox_DateTimeRange_Constructor_ReturnsSameValue()
```

Comment:
REgarding the tests naming, start with Nox I think its redundant and can be removed, does not bring any aditional value? What do you think?


Create_DateTimeRangeFrom_Creates_ValidStartAndEnd, does it seems a better name=> Scenario + TestName+ Expected?
```suggestion
    public void Create_DateTimeRangeFrom_Creates_ValidStartAndEnd()
```

Reply:
I agree, Nox prefix seems redundant. However I followed the naming from the other test classes and they all seemed to have "Nox_" prefix in the test names.
I can update my naming if you would like and see if others agree for it to be a convention.

Reply:
yes, if you can update yours would be enough, and then we can start changing others incrementally whenever it makes sense. I'm working for having this approved:https://regusit.atlassian.net/wiki/spaces/NOX/pages/3153461266/Coding+Conventions+Styles+and+Guidelines and see this one: https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#best-practices its simple and defined in .Net ... so lets do the same for now

Reply:
I've updated the naming to follow Method_Scenario_ExpectedResult as proposed in the provided link.